docs/DeveloperGuide.adoc

```diff
@@ -239,85 +238,147 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
-// tag::undoredo[]
-=== Undo/Redo feature
+
+=== Event Sorting
+
 ==== Current Implementation
 
-The undo/redo mechanism is facilitated by `VersionedAddressBook`.
-It extends `AddressBook` with an undo/redo history, stored internally as an `addressBookStateList` and `currentStatePointer`.
-Additionally, it implements the following operations:
+The sort mechanism is facilitated by Comparator<Event>.
+When the sort method for FXObservableList is called, it will take a Comparator object to be use for sorting the list.
+The comparator is able to take in two Event class objects and compare the relative parameter values.
+The parameter can be `EventName`, `Date` & `StartTime`.
+To allow sorting of these parameters, there are three types of comparators.
 
-* `VersionedAddressBook#commit()` -- Saves the current address book state in its history.
-* `VersionedAddressBook#undo()` -- Restores the previous address book state from its history.
-* `VersionedAddressBook#redo()` -- Restores a previously undone address book state from its history.
 
-These operations are exposed in the `Model` interface as `Model#commitAddressBook()`, `Model#undoAddressBook()` and `Model#redoAddressBook()` respectively.
+image::SortNewCommand1StateListDiagram.png[width="650"]
 
-Given below is an example usage scenario and how the undo/redo mechanism behaves at each step.
+image::SortSequenceDiagram.png[width="650"]
```

Comment:
Good job here, on the sequence diagram: 
- drew proper lifelines for each object
- specified invocation method
- use dashed arrows for return calls. 
- clear separation of Logic classes and Model classes